packages/suite/src/components/firmware/FirmwareInitial.tsx

```diff
@@ -152,7 +155,14 @@ export const FirmwareInitial = ({
     if (bitcoinOnlyOffer) {
         // Installing Bitcoin-only firmware in onboarding
         content = {
-            heading: <Translation id="TR_INSTALL_BITCOIN_FW" />,
+            heading: (
+                <Translation
+                    id="TR_INSTALL_BITCOIN_ONLY_FW"
+                    values={{
+                        bitcoinOnly: <Translation id="TR_FIRMWARE_TYPE_BITCOIN_ONLY" />,
```

Comment:
Not sure if it makes sense in general passing translation to another translation to preserve consistency. I guess the consistency should be ensured on different level (crowdin). and in some context it can be needed to inflect the word differently... We should discuss this with the translation company later. Feel free to keep it for now.